src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -22,30 +60,7 @@ export const Chatbox: React.FC<ChatboxProps> = ({
   sendMessage,
   deleteMessage,
 }) => {
-  const [isSendingMessage, setMessageSending] = useState(false);
-
-  // This logic dissallows users to spam into the chat. There should be a delay, between each message
-  useEffect(() => {
-    if (isSendingMessage) {
-      setTimeout(() => {
-        setMessageSending(false);
-      }, 500);
-    }
-  }, [isSendingMessage]);
-
-  const { register, handleSubmit, reset, watch } = useForm<{
-    message: string;
-  }>({
-    mode: "onSubmit",
-  });
-
-  const onSubmit = handleSubmit(({ message }) => {
-    setMessageSending(true);
-    sendMessage(message);
-    reset();
-  });
-
-  const chatValue = watch("message");
+  const [activeOption, setActiveOption] = useState<Option | null>(null);
```

Comment:
I believe this can look as simple as `useState<Option>()`, which will make it hold `Option | undefined` (we don't need to explicitly state the `undefined` in the the type of it from memory)